cypress/integration/static-root/i18n.spec.ts

```diff
@@ -27,7 +27,9 @@ describe('Localization', () => {
   it('should use Accept-Language to choose a locale', () => {
```

Comment:
```suggestion
  it.skip('should use Accept-Language to choose a locale', () => {
```

`it.skip` should work

Reply:
I thought rather than skipping entirely, I'd change it to at least test _something_, so it shows that the plugin redirects themselves are working, even if the libredirect part is broken